src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/resource/CreateInductionTest.kt

```diff
@@ -361,55 +159,20 @@ class CreateInductionTest : IntegrationTestBase() {
       )
       val createInductionEventProperties = eventPropertiesCaptor.firstValue
       assertThat(createInductionEventProperties)
-        .containsEntry("prisonId", prisonId)
+        .containsEntry("prisonId", createRequest.prisonId)
         .containsEntry("userId", dpsUsername)
         .containsKey("reference")
     }
   }
 
   @Test
   @Transactional
-  fun `should create a new induction with null collections`() {
+  fun `should create an induction for a prisoner not looking for work`() {
     // Given
     val prisonNumber = aValidPrisonNumber()
-    val createRequest = aValidCreateCiagInductionRequest(
-      hopingToGetWork = HopingToWork.YES,
-      abilityToWork = null,
-      abilityToWorkOther = null,
-      reasonToNotGetWork = null,
-      reasonToNotGetWorkOther = null,
-      workExperience = aValidCreatePreviousWorkRequest(
-        hasWorkedBefore = false,
-        workExperience = null,
-        typeOfWorkExperience = null,
-        workInterests = aValidCreateWorkInterestsRequest(
-          workInterests = null,
-          workInterestsOther = null,
-          particularJobInterests = null,
-        ),
-      ),
-      skillsAndInterests = aValidCreateSkillsAndInterestsRequest(
-        skills = null,
-        skillsOther = null,
-        personalInterests = null,
-        personalInterestsOther = null,
-      ),
-      qualificationsAndTraining = aValidCreateEducationAndQualificationsRequest(
-        educationLevel = null,
-        qualifications = null,
-        additionalTraining = setOf(TrainingType.NONE), // cannot set to null
-        additionalTrainingOther = null,
-      ),
-      inPrisonInterests = aValidCreatePrisonWorkAndEducationRequest(
-        inPrisonWork = null,
-        inPrisonWorkOther = null,
-        inPrisonEducation = null,
-        inPrisonEducationOther = null,
-      ),
-    )
+    val createRequest = aValidCreateInductionRequestForPrisonerNotLookingToWork()
```

Comment:
The ITs now reflect what the UI is likely to send (i.e.rather than attempting to send all the fields and test the mapping in one go).